### PR TITLE
[DFFS-6] Style the Contact Us Page using CSS

### PR DESCRIPTION
**Pull request by PluckyPrecious**

### CODE REVIEW

**Comment by Ireti203** [comment]
Add screenshots of the page, include screenshots for tablet and mobile screens

**Comment by PluckyPrecious** [comment]
> Add screenshots of the page, include screenshots for tablet and mobile screens

Done. 🙌🏽

File: submissions/KelechiPreciousNwachukwu/css/signup.css [hunk 1]
```diff
@@ -45,6 +45,14 @@ h2 {
     font-size: 2.5rem;
```
Comment (adaugochi) [inline submissions/KelechiPreciousNwachukwu/css/signup.css:45]: if you are using this CSS file for styling both `signup` and `contact` pages, kindly give the name of this file a generic name. the name `signup.css` kind of shows it is just for the `signup` page

**Review by adaugochi — DISMISSED** [review]

**Comment by PluckyPrecious** [comment]
Hi @Ireti203 done. @adaugochi effected changes in PR #794 thanks

**Comment by adaugochi** [comment]
kindly fix merge conflict on your PR

**Comment by PluckyPrecious** [comment]
Hi @adaugochi conflict resolved

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**